### PR TITLE
Session 11 (and crusty session 09) assignment

### PR DESCRIPTION
**Pull request by lukeatuwpce**
Apparently I forgot to submit some refactoring for session 9.  I'm not sure how to separate that into a different pull request.

### CODE REVIEW

File: students/luke/s11/iterator_1.py [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    for i in IterateMe_1():
+        print(i)
+
+    print("Testing iterator 2")
+    for i in IterateMe_2():
+        print(i)
+
+    print("Testing iterator 2 non-default parameters")
+    for i in IterateMe_2(10, 20, 2):
+        print(i)
```
Comment (davidblaszka) [inline students/luke/s11/iterator_1.py]: Good work, but can you address this 
```
Does range() behave the same?
make yours match range()
is range an iterator or an iteratable?
```

File: students/luke/s11/music_filter.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+quiet yet danceable tracks. Also, these tracks should be sorted in descending
+order by danceability so that the most danceable tracks are up top.
+"""
+
+import pandas as pd
+
+
+def make_list(dataframe):
+    return \
+        [f"{x.get('name')} by {x.get('artists')}"
```
Comment (davidblaszka) [inline students/luke/s11/music_filter.py]: This is throwing a syntax error for me. Can you fix this?

File: students/luke/s11/generator_solution.py [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+        prime = True
+        for i in range(2, val):
+            if val % i == 0:  # Not prime
+                prime = False
+                break;
+        ret = val
+        val += 1
+        if prime:
+            yield ret
+
```
Comment (davidblaszka) [inline students/luke/s11/generator_solution.py]: nice work on all these!

File: students/luke/s11/iterator_1.py (hunk 1 above)
Comment (lukeatuwpce) [reply students/luke/s11/iterator_1.py]: Apparently I didn't commit my last changes before I pushed.  I've answered these in a comment that I'll push shortly.

File: students/luke/s11/music_filter.py (hunk 2 above)
Comment (lukeatuwpce) [reply students/luke/s11/music_filter.py]: Can you elaborate?  It works for me.

edi:s11 luke$ python3 --version
Python 3.6.3
edi:s11 luke$ ./music_filter.py 
['Bad and Boujee (feat. Lil Uzi Vert) by Migos', 'Fake Love by Drake', 'HUMBLE. by Kendrick Lamar', 'Bank Account by 21 Savage', "You Don't Know Me - Radio Edit by Jax Jones"]

File: students/luke/s11/generator_solution.py (hunk 3 above)
Comment (lukeatuwpce) [reply students/luke/s11/generator_solution.py]: Thanks!

**Pull request merged**